src/Map/MapMarker.tsx

```diff
@@ -54,6 +54,8 @@ export const MapMarker: FC<Partial<MapMarkerProps>> = ({
         onMouseLeave={() => setActive(false)}
         onClick={onClick}
         tabIndex={0}
+        aria-label={typeof tooltip === 'string' ? tooltip: 'map marker'}
```

Comment:
Let's pull this logic out of the JSX element and into the function body.

Reply:
Done.